<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTANewFileCreationDialog</class>
 <widget class="QDialog" name="GTANewFileCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="FileTypeCB"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="FileNameLE"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="FilePathLE"/>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="BrowserTB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/Folder.png</normaloff>:/images/forms/img/Folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="AuthorLE"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="AuthorLB">
       <property name="text">
        <string>Author Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="MaxTimeEstimatedLE"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="MaxTimeEstimatedLB">
       <property name="text">
        <string>Max Time Estimated</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>355</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="OKPB">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/OK.png</normaloff>:/images/forms/img/OK.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="CancelPB">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/CANCEL.jpg</normaloff>:/images/forms/img/CANCEL.jpg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FileTypeCB</tabstop>
  <tabstop>FileNameLE</tabstop>
  <tabstop>AuthorLE</tabstop>
  <tabstop>FilePathLE</tabstop>
  <tabstop>BrowserTB</tabstop>
  <tabstop>OKPB</tabstop>
  <tabstop>CancelPB</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
